### DISCUSSION TITLE
Z-wave LR real world experiences?

### DISCUSSION THREAD

**Discussion by TC1666** [zwave-js/zwave-js#6956] [Feature Requests]
Long time Z-wave user, finally setup first network dedicated to LR devices. As can be seen below, I've got a Zooz ZST39 LR 800 series with the latest firmware from Zooz and a ZSE41 800-series open/close senor. I've also attached a USB extension cable to RPi4 hosting the Z-wave JS UI software and have positioned the USB stick into a position where the background noise RSSI is below -105dbm on all LR channels. 
Software is zwave-js-ui: 9.14.0 and zwave-js: 12.11.1

Yet no matter where I move the ZSE41, I can never get better than a -84dbm reading. The screen shot below is it within 1 foot of the controller. When I try moving the sensor to a longer distance to take advantage of the LR feature, it performs no better than my Z-wave plus devices connected to a 500-series controller.

What is going on here? Is Z-wave LR a farce? I've scoured the internet looking for examples of end-user experiences of LR actually living up to its promises but can find none.

<img width="995" alt="Screenshot1" src="https://github.com/zwave-js/zwave-js-ui/assets/130233920/46dbdab4-000e-47f1-8f84-a1cea0d90446">
<img width="592" alt="Screenshot2" src="https://github.com/zwave-js/zwave-js-ui/assets/130233920/f0c64b7f-a32e-43ef-a75a-36524341a2e0">
<img width="574" alt="Screenshot3" src="https://github.com/zwave-js/zwave-js-ui/assets/130233920/f1e66789-4ab1-4c2f-950c-b575de960202">

**Comment by robertsLando** [comment]
cc @AlCalzone

**Comment by AlCalzone** [comment]
Not sure what the expectation is. Do you have any actual issues with the device?

ZWLR uses dynamic power, so if it decides that transmitting at -6 dBm is enough, it will do so.
For that matter I'm not actually sure how helpful the incoming RSSI readings for ZWLR are to determine the connection strength.

**Reply by Botched1** [reply]
> For that matter I'm not actually sure how helpful the incoming RSSI readings for ZWLR are to determine the connection strength.

Agree. As the RSSI is relative to the power used it is really not important/meaningful by itself on LR transmissions. If you knew both the transmit power level and RSSI that might be meaningful, but RSSI by itself is not especially useful.

**Reply by AlCalzone** [reply]
We do actually. I'm not sure though if they can just be subtracted though.
However we don't know the transmit power of non-LR nodes, nor do we know the maximum transmit power of the ZWLR nodes, so the comparison will be lacking anyways.

**Reply by Botched1** [reply]
> We do actually. I'm not sure though if they can just be subtracted though.

Good question. I wouldn't think you could just add/subtract them though.

I was thinking more along the lines that as long as the end device was not at its maximum transmit power, then who cares what the RSSI is? However, if it is at max transmit power AND the RSSI is poor, then that would be good to know as the device may need repositioning/replacement or the hub relocated.
